server/glsp-graph/src/main/java-gen/com/eclipsesource/glsp/graph/GLayoutOptions.java

```diff
@@ -0,0 +1,266 @@
+/**
+ * Copyright (c) 2019 EclipseSource and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package com.eclipsesource.glsp.graph;
+
+import org.eclipse.emf.ecore.EObject;
+
+/**
+ * <!-- begin-user-doc -->
+ * A representation of the model object '<em><b>GLayout Options</b></em>'.
+ * <!-- end-user-doc -->
+ *
+ * <p>
+ * The following features are supported:
+ * </p>
+ * <ul>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getPaddingLeft <em>Padding Left</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getPaddingRight <em>Padding Right</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getPaddingTop <em>Padding Top</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getPaddingBottom <em>Padding Bottom</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getPaddingFactor <em>Padding Factor</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#isResizeContainer <em>Resize Container</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getVGap <em>VGap</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getHGap <em>HGap</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getVAlign <em>VAlign</em>}</li>
+ *   <li>{@link com.eclipsesource.glsp.graph.GLayoutOptions#getHAlign <em>HAlign</em>}</li>
+ * </ul>
+ *
+ * @see com.eclipsesource.glsp.graph.GraphPackage#getGLayoutOptions()
+ * @model
+ * @generated
+ */
+public interface GLayoutOptions extends EObject {
```

Comment:
Getters for minimal width and height are missing here.
(https://github.com/eclipse/sprotty-server/pull/26/commits/f8fd447a07a155dfab61920ed8879b6a64a7adb9#diff-450e8042bd338dc9004c79c10c7ff07d)